backend/medtagger/repositories/labels.py

```diff
@@ -24,25 +24,22 @@ def get_label_by_id(label_id: LabelID, fetch_binary_masks: bool = False) -> Labe
         with db_session() as session:
             label = session.query(Label).filter(Label.id == label_id).one()
         if fetch_binary_masks:
-            label = LabelsRepository._fetch_label_selections_binary_masks(label)
+            label = LabelsRepository._fetch_label_elements_binary_masks(label)
         return label
 
     @staticmethod
-    def get_random_label(status: LabelStatus = None, fetch_binary_masks: bool = False) -> Label:
+    def get_random_label(fetch_binary_masks: bool = False) -> Label:
```

Comment:
We still need to have status on Label level. Doctors would like to see only unseen/not validated Labels for verification. Right now, they will get random one which may result in double verification!